config/300-decorator.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2019 The Knative Authors
```

Comment:
Copyright 2019 Google LLC

Reply:
I have been transitioning back to Knative Authors